src/components/list/NoResults.tsx

```diff
@@ -14,7 +14,7 @@ export enum NoResultsType {
 
 export const NoResults = ({ onL2, type }: { onL2?: boolean; type: NoResultsType }) => {
   const { colors } = useTheme();
-  const { data: assetCount } = useUserAssetCount();
+  const assetCount = useUserAssetsStore(state => state.legacyUserAssets.length);
```

Comment:
this doesn't need to be `legacyUserAssets` if we're just using the count right? since it should be the same as the non-legacy version?

Reply:
i'm gonna do option 3

Reply:
You should be able to use `.size` on a map in place of `.length`

Reply:
you're BACK!